packages/at_secondary_server/lib/src/verb/handler/abstract_update_verb_handler.dart

```diff
@@ -164,8 +177,7 @@ abstract class AbstractUpdateVerbHandler extends ChangeVerbHandler {
       metadata.ttb = AtMetadataUtil.validateTTB(verbParams[AtConstants.ttb]);
     }
     if (verbParams[AtConstants.ttr] != null) {
-      metadata.ttr =
-          AtMetadataUtil.validateTTR(int.parse(verbParams[AtConstants.ttr]!));
+      metadata.ttr = hu.validateTTR(int.parse(verbParams[AtConstants.ttr]!));
```

Comment:
Using local impl of validateTTR in order to resolve a bug. PR will follow for at_utils to resolve in the same way